src/ert/gui/simulation/ensemble_smoother_panel.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, ert: EnKFMain, notifier: ErtNotifier):
         facade = LibresFacade(ert)
         layout = QFormLayout()
 
-        self._case_selector = CaseSelector(facade, notifier)
+        self._case_selector = CaseSelector(facade, notifier, show_create_new_case=True)
```

Comment:
Shouldnt the `show_create_new_case` property be removed? Not sure it is related to this PR, but if I now run two experiments, I get two ensembles, both named `default`. If I combine this PR with: https://github.com/equinor/ert/pull/4982 i only get one ensemble as I would expect, so perhaps the property is redundant?

Reply:
I have removed it.